Swift/Tests/CBLTestHelper.m

```diff
@@ -27,4 +27,18 @@ + (void) allowExceptionIn: (void (^)(void))block {
     --gC4ExpectExceptions;
 }
 
++ (BOOL) catchException: (void(^)(void))tryBlock error: (__autoreleasing NSError **)error {
```

Comment:
I have never specified __autoreleasing. What doesn't it do specifically for this case?

Reply:
`NSError** error` implicitly qualified as `__autoreleasing `. For better clarity, the example I copied, have explicitly specified the `__autoreleasing`. I will remove it. 

https://stackoverflow.com/questions/13587742/nserror-and-autoreleasing/13590696#13590696
https://clang.llvm.org/docs/AutomaticReferenceCounting.html#indirect-parameters